plugins/salesforce-as-datasource-as-eventsource/README.md

```diff
@@ -1 +1,146 @@
+
+## Steps to use Salesforce plug-in in godspeed framework:
+
+## How to Use
+- Create a godspeed project from the CLI , open the created project in vscode and then add the plugin from the CLI of vscode, select the `@godspeedsystems/plugins-salesforce-as-datasource-as-eventsource` to integrate the plugin.
+
+```
+> godspeed plugin add
+       ,_,   ╔════════════════════════════════════╗
+      (o,o)  ║        Welcome to Godspeed         ║
+     ({___}) ║    World's First Meta Framework    ║
+       " "   ╚════════════════════════════════════╝
+? Please select godspeed plugin to install: (Press <space> to select, <Up and Down> to move rows)
+┌──────┬────────────────────────────────────┬────────────────────────────────────────────────────────────────────┐
+│      │ Name                               │ Description                                                        │
+├──────┼────────────────────────────────────┼────────────────────────────────────────────────────────────────────┤
+│  ◯   │ prisma-as-datastore                │ Prisma as a datasource plugin for Godspeed Framework.              │
+├──────┼────────────────────────────────────┼────────────────────────────────────────────────────────────────────┤
+│  ◯   │ aws-as-datasource                  │ aws as datasource plugin for Godspeed Framework                    │
+├──────┼────────────────────────────────────┼────────────────────────────────────────────────────────────────────┤
+│  ◯   │ excel-as-datasource                │ excel as datasource plugin for Godspeed Framework                  │
+├──────┼────────────────────────────────────┼────────────────────────────────────────────────────────────────────┤
+│  ◯   │ mailer-as-datasource               │ mailer as datasource plugin for Godspeed Framework                 │
+├──────┼────────────────────────────────────┼────────────────────────────────────────────────────────────────────┤
+│ ❯◯   │ salesforce-as-datasource-as-eventsource │ salesforce as datasource-as-eventsource plugin for Godspeed Framework   │
+└──────┴────────────────────────────────────┴────────────────────────────────────────────────────────────────────┘
+```
+
+### Example usage Datasource (Producer):
+
+1. Update configuration file based on your requirements in `Datasource/salesforce.yaml`.
+#### salesforce config ( src/datasources/salesforce.yaml )
+```yaml
+type: salesforce
+loginUrl: 'https://dummy.salesforce.com'
+username: dummy.username.password@organizationname.com
+password: password@123
+```
+
+
+
+#### salesforce event for Producer ( src/events/salesforce_pub.yaml )
+In the event, we establish an HTTP endpoint that accepts parameters such as message content. When this endpoint is invoked, it triggers the `salesforce-publish` function. This function, in turn, takes the provided  message as input arguments and performs the task of publishing the message to the specified salesforce api end-point.
```

Comment:
Why are we not telling first that you can call any salesforce endpoint now, and this is the generic way to do it as shown in the following example event triggering a call to a salesforce api